targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K64F/TARGET_FRDM/PeripheralPins.c

```diff
@@ -16,6 +16,123 @@
 
 #include "PeripheralPins.h"
 
+/************GPIO***************/
+const PinMap PinMap_GPIO[] = {
+    {PTA0, GPIO_A, 1},
+    {PTA1, GPIO_A, 1},
+    {PTA2, GPIO_A, 1},
+    {PTA3, GPIO_A, 1},
+    {PTA4, GPIO_A, 1},
+    {PTA5, GPIO_A, 1},
+    {PTA6, GPIO_A, 1},
+    {PTA7, GPIO_A, 1},
+    {PTA8, GPIO_A, 1},
+    {PTA9, GPIO_A, 1},
+    {PTA10, GPIO_A, 1},
+    {PTA11, GPIO_A, 1},
+    {PTA12, GPIO_A, 1},
+    {PTA13, GPIO_A, 1},
+    {PTA14, GPIO_A, 1},
+    {PTA15, GPIO_A, 1},
+    {PTA16, GPIO_A, 1},
+    {PTA17, GPIO_A, 1},
+    {PTA18, GPIO_A, 1},
+    {PTA19, GPIO_A, 1},
+    {PTA24, GPIO_A, 1},
+    {PTA25, GPIO_A, 1},
+    {PTA26, GPIO_A, 1},
+    {PTA27, GPIO_A, 1},
+    {PTA28, GPIO_A, 1},
+    {PTA29, GPIO_A, 1},
+
+    {PTB0, GPIO_B, 1},
+    {PTB1, GPIO_B, 1},
+    {PTB2, GPIO_B, 1},
+    {PTB3, GPIO_B, 1},
+    {PTB4, GPIO_B, 1},
+    {PTB5, GPIO_B, 1},
+    {PTB6, GPIO_B, 1},
+    {PTB7, GPIO_B, 1},
+    {PTB8, GPIO_B, 1},
+    {PTB9, GPIO_B, 1},
+    {PTB10, GPIO_B, 1},
+    {PTB11, GPIO_B, 1},
+    {PTB12, GPIO_B, 1},
+    {PTB13, GPIO_B, 1},
+    {PTB16, GPIO_B, 1},
+    {PTB17, GPIO_B, 1},
+    {PTB18, GPIO_B, 1},
+    {PTB19, GPIO_B, 1},
+    {PTB20, GPIO_B, 1},
+    {PTB21, GPIO_B, 1},
+    {PTB22, GPIO_B, 1},
+    {PTB23, GPIO_B, 1},
+
+    {PTC0, GPIO_C, 1},
+    {PTC1, GPIO_C, 1},
+    {PTC2, GPIO_C, 1},
+    {PTC3, GPIO_C, 1},
+    {PTC4, GPIO_C, 1},
+    {PTC5, GPIO_C, 1},
+    {PTC6, GPIO_C, 1},
+    {PTC7, GPIO_C, 1},
+    {PTC8, GPIO_C, 1},
+    {PTC9, GPIO_C, 1},
+    {PTC10, GPIO_C, 1},
+    {PTC11, GPIO_C, 1},
+    {PTC12, GPIO_C, 1},
+    {PTC13, GPIO_C, 1},
+    {PTC14, GPIO_C, 1},
+    {PTC15, GPIO_C, 1},
+    {PTC16, GPIO_C, 1},
+    {PTC17, GPIO_C, 1},
+    {PTC18, GPIO_C, 1},
+    {PTC19, GPIO_C, 1},
+
+    {PTD0, GPIO_D, 1},
+    {PTD1, GPIO_D, 1},
+    {PTD2, GPIO_D, 1},
+    {PTD3, GPIO_D, 1},
+    {PTD4, GPIO_D, 1},
+    {PTD5, GPIO_D, 1},
+    {PTD6, GPIO_D, 1},
+    {PTD7, GPIO_D, 1},
+    {PTD8, GPIO_D, 1},
+    {PTD9, GPIO_D, 1},
+    {PTD10, GPIO_D, 1},
+    {PTD11, GPIO_D, 1},
+    {PTD12, GPIO_D, 1},
+    {PTD13, GPIO_D, 1},
+    {PTD14, GPIO_D, 1},
+    {PTD15, GPIO_D, 1},
+
+    {PTE0, GPIO_E, 1},
+    {PTE1, GPIO_E, 1},
+    {PTE2, GPIO_E, 1},
+    {PTE3, GPIO_E, 1},
+    {PTE4, GPIO_E, 1},
+    {PTE5, GPIO_E, 1},
+    {PTE6, GPIO_E, 1},
+    {PTE7, GPIO_E, 1},
+    {PTE8, GPIO_E, 1},
+    {PTE9, GPIO_E, 1},
+    {PTE10, GPIO_E, 1},
+    {PTE11, GPIO_E, 1},
+    {PTE12, GPIO_E, 1},
+    // {PTE24, GPIO_E, 1}, // fixed pull-up (for I2C)
+    // {PTE25, GPIO_E, 1}, // fixed pull-up (for I2C)
```

Comment:
This only applies to the pull up/down tests right? If so is there a way the GPIO tests could run all its normal testing on these pins except the pull up/down test?